parts/swarmmastervars.t

```diff
@@ -117,10 +117,15 @@
     "osImageSKU": "14.04.4-LTS", 
 {{end}}
     "osImageVersion": "latest", 
+{{if GetLocation}}        
```

Comment:
I would like to see this code in the parameters.  Something like:

    "location": {
{{if GetLocation}}
	"defaultValue": "{{GetLocation}}",
{{else}}
    "defaultValue": "",
{{end}}
      "metadata": {
        "description": "Sets the location for all resources in the cluster."
      },
      "type": "string"
    },

Then the variables would be (using series formula mod(n+2,n+1), to achieve the correct selection)):

"locationIndex": "[mod(length(variables('location'))+2,length(variables('location'))+1)]",
"locations": [
"[resourceGroup().location]",
"[variables('location')]"
],
"location": "[variables('locations')[variables('locationIndex')]]"